install-and-configure/advanced-configuration/eks-optimized/kubecost-aws-distro-open-telemetry.md

```diff
@@ -0,0 +1,139 @@
+# Multi-Cluster Kubecost with AWS Distro for OpenTelemetry (ADOT)
+
+## See also
+
+* [AMP Overview](/install-and-configure/advanced-configuration/eks-optimized/aws-amp-integration.md)
+* [AWS Agentless AMP](/install-and-configure/advanced-configuration/eks-optimized/kubecost-agentless-amp.md)
+* [AMP with remote_write](/install-and-configure/advanced-configuration/eks-optimized/amp-with-remote-write.md)
+
+## Overview
+
+This guide will walk you through the steps to deploy Kubecost with [AWS Distro for Open Telemetry (ADOT)](https://aws-otel.github.io/) to collect metrics from your Kubernetes clusters utilizing the `EKS-Optimized` license.
+
+* Note: Kubecost `EKS-Optimized` allows for 15 days of query history. Unlock unlimited history with a [Kubecost subscription](https://www.kubecost.com/pricing).
+
+![Architecture Diagram](/images/adot-architecture.png)
+
+## Prerequisites
+
+1. An AWS Managed Prometheus Workspace is required to use ADOT

```

Comment:
I don't like how this is worded. When we say "Kubecost subscription," we should be specifically saying "Kubecost Enterprise, correct?